<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>459</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>459</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>459</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="minimumSize">
       <size>
        <width>270</width>
        <height>400</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>400</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="mainPage">
       <widget class="QGroupBox" name="mainPage_groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>271</width>
          <height>400</height>
         </rect>
        </property>
        <property name="title">
         <string>Main Menu</string>
        </property>
        <widget class="QPushButton" name="m1Home">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string> POPUP TEST</string>
         </property>
        </widget>
        <widget class="QPushButton" name="settingsButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
        <widget class="QPushButton" name="lockButton">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>170</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Lock</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lockStatus">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>210</y>
           <width>113</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Lock Inactive</string>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="settingsPage">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>250</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <widget class="QGroupBox" name="settingsPage_groupBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>271</width>
          <height>400</height>
         </rect>
        </property>
        <property name="title">
         <string>Settings</string>
        </property>
        <widget class="QLabel" name="NotificationStrength">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>271</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Notification Strength</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QSlider" name="horizontalSlider">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>231</width>
           <height>22</height>
          </rect>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
        <widget class="QLabel" name="NotificationType">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>80</y>
           <width>271</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Notification Type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QListWidget" name="listWidget">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>100</y>
           <width>231</width>
           <height>181</height>
          </rect>
         </property>
        </widget>
        <widget class="QPushButton" name="back_page_Button">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>310</y>
           <width>75</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
